headers/types/common/common.h

```diff
@@ -1213,4 +1213,32 @@ ASSERT_SIZE(struct thread_info, 52); // Could be shorter
 // TODO: Add more data file structures, as convenient or needed, especially if the load address
 // or pointers to the load address are known.
 
+// State of the screen fade in all other modes except dungeon mode
+#pragma pack(push, 1)
+struct screen_fade {
+    undefined field0_0x0;
+    undefined field1_0x1;
+    undefined field2_0x2;
+    undefined field3_0x3;
+    enum fade_status status;
+    int remaining_frames;
+    // Initial duration of the fade in frames
+    int duration;
+    // What brightness the fade ends at. 256 means fade to white, -256 means fade to black
+    short target_delta_brightness;
+    // Max absolute value of the brightness during the fade. 256 means a full fade
+    short max_brightness;
+    // Current brightness of the fade. Positive numbers turn the screen white, negative to black
+    short delta_brightness;
+};
+#pragma pack(pop)
+ASSERT_SIZE(struct screen_fade, 22);
+
+// The file timer. Counts every frame spent on the file, except loading, lag, saving and being
+// in the main menu.
+struct play_time {
+    uint32_t seconds;
+    uint8_t frames;
```

Comment:
Should have field offsets in comments, as explained before.

Reply:
I'm actually kind of inconsistent about enforcing this lol. I do always prefer having offsets, and with your PRs for example I'm probably more strict because I know you're a consistent contributor and don't mind tidying things up. With others sometimes I'm more lazy/lenient in favor of making things easier 😛.

I'd say that's my general reviewing philosophy for various other minor things as well.